app/src/main/java/com/example/desafioandroid/util/SharedPreferences.java

```diff
@@ -0,0 +1,24 @@
+package com.example.desafioandroid.util;
+
+import android.content.Context;
+
+public class SharedPreferences {
+    private android.content.SharedPreferences preferences;
+    private android.content.SharedPreferences.Editor editor;
+    private static final String NOME_ARQUIVO = "solutis.desafiofilmeandroid";
+    private final String CHAVE_ID_MOVIE = "movie_id";
+
+    public SharedPreferences(Context context) {
+        this.preferences = context.getSharedPreferences(NOME_ARQUIVO, Context.MODE_PRIVATE);
+        editor = preferences.edit();
```

Comment:
O edit() só deve ser chamado quando uma alteração for feita e depois seguido de uma chamada de commit() ou apply()